go.mod

```diff
@@ -3,10 +3,7 @@ module github.com/ethereum/go-ethereum
 go 1.15
```

Comment:
shouldn't the module be `consensys/quorum` ? I think that's one of the main issues because otherwise it will not appear when doing a `go get`

Reply:
that's why I asked before if this will solve the imports.

I think to properly solve it, we need to change the path here and the path on the code imports.